Kaptain 2.0.0 Docs monitoring folder (#4316)
mesosphere/dcos-docs-site: pages/dkp/kaptain/2.0.0/monitoring/index.md

---
layout: layout.pug
navigationTitle: Monitoring
title: Monitoring
excerpt: Monitor ML-workloads and resources utilization in Kaptain
menuWeight: 12
---

Monitoring is one of the most important aspects of any machine learning workflow. Kaptain Dashboard is a single pane

> of glass allowing Kaptain users (e.g., data scientists) to see the current status of their machine learning workloads

Author: ```suggestion
of glass allowing Kaptain users (for example data scientists) to see the current status of their machine learning workloads
```

and how many resources they are consuming. With the dashboard, users can observe the state of their ML workloads without
having to piece together information from various sources.

## Kaptain Dashboard

To access the dashboard application, click on a "Dashboard" link in the Kubeflow left sidebar.
The "Kaptain Dashboard" page consists of four sections. To select a time range for graphs, use the "Time Period" control
on the top-left corner.

The top section contains an overview of the current health of Kaptain components.

![Components](img/components.png)

The "Data" section provides the information about what is currently running in a user's namespace; graphs show the
number of active notebooks, pipelines, ML experiments, and trials:

![Data](img/data.png)

> In the "Jobs" section, users can get the current state of machine learning jobs, e.g., how many `TFJob` or `PytorchJob`

Author: ```suggestion
In the "Jobs" section, users can get the current state of machine learning jobs, for example how many `TFJob` or `PytorchJob`
```

resources were created, completed, or failed:

![Jobs](img/jobs.png)

The "System Resources" section is all about resource consumption. The graphs show how many resources are being used

> by workloads in the user's namespace. There are three graphs for each type of resource:

Author: ```suggestion
by workloads in the user's namespace. Kaptain has three graphs for each type of resource:
```
